ContentManagerPlugin type not exported from @strapi/content-manager/strapi-admin

### Node Version

22.18.0

### NPM/Yarn/PNPM Version

10.9.3

### Strapi Version

5.20.0

### Operating System

MacOS

### Database

SQLite

### Javascript or Typescript

Typescript

### Reproduction URL

_No response_

### Bug Description

The [documentation mentions](https://docs.strapi.io/cms/plugins-development/content-manager-apis#api-shape) you can `import type { ContentManagerPlugin } from '@strapi/content-manager/strapi-admin';` but `ContentManagerPlugin` type is not exported.



### Steps to Reproduce

1. Create a Strapi plugin that integrates with content-manager
2. Try to import the ContentManagerPlugin type:
     ```typescript
     import type { ContentManagerPlugin } from '@strapi/content-manager/strapi-admin'
3. TypeScript error: Cannot find module '@strapi/content-manager/strapi-admin' or its corresponding type declarations

### Expected Behavior

Should be able to import ContentManagerPlugin type for proper TypeScript typing when working with app.getPlugin('content-manager').

### Logs

```shell

```

### Code Snippets

From the [documentation](https://docs.strapi.io/cms/plugins-development/content-manager-apis#api-shape):

```
import type { ContentManagerPlugin } from '@strapi/content-manager/strapi-admin';

const apis =
  app.getPlugin('content-manager').apis as ContentManagerPlugin['config']['apis'];

apis.addEditViewSidePanel([ReleasesPanel]);
```

### Media

_No response_

### Additional information

_No response_

### Confirmation Checklist

- [x] I have checked the existing [issues](https://github.com/strapi/strapi/issues) for duplicates.
- [x] I agree to follow this project's [Code of Conduct](https://github.com/strapi/strapi/blob/develop/CODE_OF_CONDUCT.md).

--- a/packages/core/content-manager/admin/src/exports.ts
+++ b/packages/core/content-manager/admin/src/exports.ts
@@ -27,6 +27,7 @@ export type {
   BulkActionComponent,
   BulkActionComponentProps,
   BulkActionDescription,
+  ContentManagerPlugin,
   DescriptionComponent,
   DescriptionReducer,
   PanelComponentProps,

